README.md

```diff
@@ -27,6 +27,8 @@ _For comparison to other tools including pipsi, see
 
 ## Install pipx
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/pipx.svg?columns=3&exclude_unsupported=1)](https://repology.org/metapackage/pipx/versions)
```

Comment:
How about putting it in a collapsible section so that it occupies less space?

```suggestion
<details>
  <summary>Packaging status</summary>
  
  [![Packaging status](https://repology.org/badge/vertical-allrepos/pipx.svg?columns=3&exclude_unsupported=1)](https://repology.org/metapackage/pipx/versions)
</details>
```

Reply:
Seeing this I'd be +1 on not having it collapsed. It was not taking up that much space in the first place with 3 columns, and I think it being in a collapsible section makes it very hard to find, to the point where I think most people won't find it at all.

WDYT @dukecat0?

Reply:
Makes sense to me, but do you prefer moving it to `installation.md` instead? I believe that's the place for a more detailed installation instruction.

Reply:
If the shields work there that'd be fine for me. 

With the current setup, it's broken on the [main page of the docs](https://pipx--1276.org.readthedocs.build/1276/). @sstandre, can you please have a look at fixing that? It's probably because of the `<details>` section, so might be as simple as just moving the shield to the `installation.md` and not having it collapsible. 

> Or in the future, we should move some of the instructions from `README.md` to `installation.md` so we don't need to update both of them when there are updates.

I like the idea of uncluttering the main README :+1: